src/main/java/net/openhft/chronicle/bytes/internal/ChunkedMappedBytes.java

```diff
@@ -256,10 +256,10 @@ protected void writeCheckOffset(final long offset, final long adding)
             throws BufferOverflowException, IllegalStateException {
 
         throwExceptionIfClosed();
-        if (offset < 0 || offset > mappedFile.capacity() - adding)
+        if (offset + adding < start() || offset > mappedFile.capacity() - adding)
```

Comment:
was this a bug to be using `0` instead of `start()` previously?

Reply:
Yes I think so